tests/jaxley_identical/test_grad.py

```diff
@@ -33,13 +33,13 @@ def test_network_grad():
     _ = np.random.seed(0)
     pre = net.cell([0, 1, 2])
     post = net.cell([3, 4, 5])
-    fully_connect(pre, post, IonotropicSynapse())
-    fully_connect(pre, post, TestSynapse())
+    fully_connect(pre, post, IonotropicSynapse(), random_post_comp=True)
+    fully_connect(pre, post, TestSynapse(), random_post_comp=True)
```

Comment:
is there a test for `random_post_comp=False`? At least for `fully_connect`, but also for the others? At least to check that they do not break?

Reply:
All of the tests in test_connection.py are with random_post_comp=False, but the tests that used connection.py everywhere else (test_grad.py and test_basic_modules.py) use random_post_comp=True with fully connect (so that the simulation results are the same as before). I could add tests for random_post_comp=True to test_connection.py -- would this then be enough coverage?